sql/src/main/java/io/crate/planner/consumer/QueryThenFetchConsumer.java

```diff
@@ -119,6 +119,13 @@ public PlannedAnalyzedRelation visitQueriedTable(QueriedTable table, ConsumerCon
                         collectSymbols.add(symbol);
                     }
                 }
+
+                MergeProjection mergeProjection = new MergeProjection(
+                        collectSymbols,
```

Comment:
btw: would make sense to use the OrderBy object in the future, since it is now streamable